### PR TITLE
Copy updates for Download VA Letters

### PR DESCRIPTION
**Pull request by rroueche**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3701

### CODE REVIEW

File: src/js/letters/containers/VeteranBenefitSummaryLetter.jsx [hunk 1]
```diff
@@ -78,26 +78,25 @@ class VeteranBenefitSummaryLetter extends React.Component {
...
         {militaryServiceInfo}
         <h2>VA Benefit Information</h2>
+        <p>
+          Select which statements you want to include in your letter.
+        </p>
         {vaBenefitInformation}
         <p>
           If you see incorrect information for service periods or disability status,
-          please send a question using VA's <a href="/">Inquiry Routing & Information
+          please send a question using VA's <a href="https://iris.custhelp.com/app/ask">Inquiry Routing & Information
```
Comment (annekainicUSDS) [inline src/js/letters/containers/VeteranBenefitSummaryLetter.jsx]: Can we please also add the property `target="_blank"` to this `<a>`? For an example of what this should look like, see above for the address update link.

**Review by annekainicUSDS — COMMENTED** [review]
Looks good! Just one minor comment.

File: src/js/letters/containers/VeteranBenefitSummaryLetter.jsx (hunk 1 above)
Comment (rroueche) [inline src/js/letters/containers/VeteranBenefitSummaryLetter.jsx]: oh yeah most def one sec

Comment (rroueche) [inline src/js/letters/containers/VeteranBenefitSummaryLetter.jsx]: ok done! lmk if all good now @annekainicUSDS

**Review by annekainicUSDS — APPROVED** [review]
LGTM!

**Pull request merged**